data/management/commands/import_mrs_data.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management.base import BaseCommand
+
+from data.mrs import fetch_mrs, import_mr
+
+
+class Command(BaseCommand):
+    help = 'Import mrs opened by newcomers'
+
+    COLLECTIONS = staticmethod(fetch_mrs)
+    IMPORT_DATA = staticmethod(import_mr)
+
+    def handle(self, *args, **options):
+        # for data in self.COLLECTIONS('github'):
```

Comment:
I had a similar doubt - Wouldn't these already be empty? (hence code being functional if this is not commented)

Reply:
Yes, I would have to add an `if` condition for it to work on the empty data.
Otherwise, we would get the error:
```
TypeError: 'NoneType' object is not iterable
```

Reply:
But I think it would be good to left it commented because the data at https://webservices.coala.io/mrs/gitlab/all is not empty, when we go to that url, it just keeps loading, which lead to our `import command` hanging for a while.

I will un-comment these when we get some data from the webservices.